core/registry.go

```diff
@@ -125,6 +125,7 @@ func (r *Registry) GetManifest(token string) (*Manifest, error) {
 
 	req.Header.Set("User-Agent", "abroot")
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
+	req.Header.Set("Accept", "application/vnd.oci.image.manifest.v1+json")
```

Comment:
```suggestion
	req.Header.Set("Accept", "application/vnd.oci.image.manifest.v2+json")
```

Shouldn't this be v2?

If I remember correctly the current version of Docker OCI manifest GHCR uses is v2.

Reply:
no, v2 doesnt get accepted

Reply:
Just now referenced https://github.com/opencontainers/image-spec/blob/main/media-types.md, seems like I indeed confused the OCI media type (which is still in v1) with Docker one (which is in v2 format).

Resolving the conversation.